internal/revproxy/main.go

```diff
@@ -54,8 +54,13 @@ func (r *Revproxy) RegisterHandlers(e *echo.Echo, commonMiddlewares ...echo.Midd
 	sk := e.Group("/api/data/user/secret_key", commonMiddlewares...)
 	sk.GET("/", echo.NotFoundHandler)
 
+	if r.config.EnableV1Services && !r.config.EnableInternalGitlab {
+		panic("cannot support turnign")
+	}
+
 	// Middlewares and routing is configured depending on `EnableV1Services`
-	if r.config.EnableV1Services {
+	if r.config.EnableV1Services && r.config.EnableInternalGitlab {
```

Comment:
I think it would be clearer to have the three cases in this order:
1. `if r.config.EnableV1Services {`
2. `} else if r.config.EnableInternalGitlab {`
3. `} else {`

Reply:
addressed in https://github.com/SwissDataScienceCenter/renku-gateway/pull/772/commits/54d2db1ede958e5d2971ce222e3ab05c3bbd2b17